### ISSUE TITLE
Don't mount whole host drive ( "/") into containers

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud-docker#786]
**Is your feature request related to a problem? Please describe.**
Mounting the whole host drive into `utils`, causes issues on several operating systems and is not great from a security perspective. We are currently doing this to be able to check availability of the backup path on start of utils and prompt user to change if not available.

**Describe the solution you'd like**
- mount backup path only *after* user entered backup path in flow
- mount .xud-docker path and potentially different `network` paths as per `xud-docker.conf` in subsequent step

**Comment by reliveyy** [comment]
> mount backup path only after user entered backup path in flow

@kilrau You cannot append new volumes into a running contianer thus why we need to mount "/" into utils.

**Comment by kilrau** [comment]
Aware, so there will be container restarts needed whenever something like `backup` path or `network` path changes.

**Comment by reliveyy** [comment]
> Aware, so there will be container restarts needed whenever something like `backup` path or `network` path changes.

Yes. But we also map "/" into xud container too. So there is no restart required for xud in the past.
